hub/compute/pipeline.py

```diff
@@ -0,0 +1,95 @@
+import hub
+
+import ray
+
+from hub.utils import batch
+
+
+class Transformer:
```

Comment:
Please change the name Transformer to Transform, it might be confusing with Transformer Neural Network Architecture.

Reply:
This is internal class, we can name it whatever we want. Should be have a naming scheme to seperate api from our internal tools?

Reply:
Even if it is internal if others would be interested in contributing it might be confusing.